templates/sync-meeting-agenda.md

```diff
@@ -0,0 +1,11 @@
+# Sync Meeting Agenda Template
+
+- Check-in
+- Assign roles
+- Turn on recording
+- Consent to agenda
+- Agenda Item 1 (*Champion's Name* **Time**: $N Minutes)
+    - **Goal**:  
+    - **Proposal**:  
```

Comment:
We should also link to any previous discussion on the topic so that folks can get context quickly.